### PR TITLE
Remove CfS and add community survey for a program. and...

### PR DESCRIPTION
**Pull request by KMR-zoar**
Remove CfS and add community survey for a program.
Remove link to scholarship in nav and disable link to scholarship in footer.

issue #42

### CODE REVIEW

**Comment by KMR-zoar** [comment]
PR including change for JA.
I am glad that would you please review the English expression method.

**Comment by KMR-zoar** [comment]
Community survey for program was closed.

File: _includes/announce.html [hunk 1]
```diff
@@ -7,13 +7,9 @@ <h2 class="dark">Last chance - Apply now!</h2>
    </div>
    <div class="col12 clearfix contain center space-bottom8">
       <div class="limiter">
-         <div class="col6 pad0y space-bottom2">
-            <h3 class="announce-title"><a href="https://blog.openstreetmap.org/2017/02/16/propose-your-session-to-state-of-the-map-2017/" class="sub_color">Call for Proposals</a></h3>
-            <p class="varela">Please submit your session proposals for a chance to be included in the program.</p>
-         </div>
-         <div class="col6 pad0y space-bottom2">
-	         <h3 class="announce-title"><a href="https://blog.openstreetmap.org/2017/02/22/apply-for-a-scholarship-to-state-of-the-map-2017/" class="sub_color">Apply for a Scholarship</a></h3>
-            <p class="varela">Thanks to the support of our sponsors, scholarships will help bring us together.</p>
```
Comment (althio) [inline _includes/announce.html]: <div class="col12 clearfix contain center space-bottom8">
      <div class="limiter">
         <div class="col6 pad0y space-bottom2">
            <h3 class="announce-title">Conference program</h3>
            <p class="varela">Thank you for the great responses to our <a href="https://blog.openstreetmap.org/2017/02/16/propose-your-session-to-state-of-the-map-2017/" class="sub_color">call for proposals</a> and <a href="https://blog.openstreetmap.org/2017/04/11/community-survey-for-the-state-of-the-map-2017-program/" class="sub_color">community input</a> for building the program!</p>
         </div>
         <div class="col6 pad0y space-bottom2">
            <h3 class="announce-title">Scholarship program</h3>
            <p class="varela">Thanks to the support of our sponsors, scholarships will help bring us together. Thank you to all the great people <a href="https://blog.openstreetmap.org/2017/02/22/apply-for-a-scholarship-to-state-of-the-map-2017/" class="sub_color">who applied for a scholarship</a>!</p>
         </div>
      </div>
   </div>

File: program/index.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ title: Program
 titlecontent:
 ---
 
-State of the Map 2017 is taking place from **August 18th to 20th 2017** in Aizuwakamatsu, Japan. We are looking forward to designing a program with you - the OpenStreetMap community. Please <a href="https://blog.openstreetmap.org/2017/02/16/propose-your-session-to-state-of-the-map-2017/">submit your session proposals</a> for a chance to be included in the program.
```
Comment (althio) [inline program/index.md:7]: State of the Map 2017 is taking place from **August 18th to 20th 2017** in Aizuwakamatsu, Japan. We are looking forward to designing a program with you - the OpenStreetMap community. Thank you for the great responses to our <a href="https://blog.openstreetmap.org/2017/02/16/propose-your-session-to-state-of-the-map-2017/">call for proposals</a> and <a href="https://blog.openstreetmap.org/2017/04/11/community-survey-for-the-state-of-the-map-2017-program/">community input</a>! The final selection of talks and the building of the detailed program are making good progress thanks to our volunteers in the program committee. Watch here for more information soon.

**Review by althio — COMMENTED** [review]
I commented / reviewed some parts to be at the current state.
Please test formatting @KMR-zoar and/or review English wording @RobJN @mikelmaron @nomoregrapes

**Comment by mikelmaron** [comment]
@althio your edits look fine -- what are we blocking on now? @KMR-zoar making edits on this PR?

**Comment by KMR-zoar** [comment]
@mikelmaron I edit now. wait a minute.

**Comment by KMR-zoar** [comment]
@mikelmaron I edited this PR from great comment.
Created new commit and merged.

@althio Thanks for your comment!

**Pull request merged**